javascript/javascript2/week1/homework.js

```diff
@@ -0,0 +1,45 @@
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+
+function shortestWord(words){
+    let res 
+    for(let i=0;i<words.length;i++){
+    for(let j=0;j<words.length;j++){
+        if(i!=j){
+        if( words[i].length<words[j].length){
+            res = words[i]
+        }
+        else{
+            res =words[j]
+        }}
+    }
+}
+return res;
+
+}
+shortestWord(danishWords);
+danishWords.reduce((curWord,nextWord) => curWord.length<nextWord.length? curWord : nextWord)
```

Comment:
this looks like a much more readable and short solution to me than two nested for loops

Reply:
yes... i wrote for loop and tried with reduce